webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTable.js

```diff
@@ -15,10 +15,13 @@ const ModelsTable = ({
   error,
   status,
   results,
+  itemCount,
+  pagination,
+  onPaginationChange,
 }) => {
   useEffect(() => {
     getTableItems(getURIQuery(window.location.href));
-  }, []);
+  }, [pagination.page, pagination.perPage]);
```

Comment:
I found out re-running the hook when `perPage` changes this still causes the the double calls under certain conditions:

1) set 'Entries per page' under Settings -> General tab to a non-default value, 50 will work
2) refresh the models page

The initial value is 20, but it gets updated after the API call finishes, which triggers another call.